apps/web/src/components/TrendingSuggestions/TrendingSuggestions.test.tsx

```diff
@@ -0,0 +1,80 @@
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { render, waitFor, screen } from '@testing-library/react';
+import TrendingSuggestions from './TrendingSuggestions';
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+const queryClient = new QueryClient();
+
+const server = setupServer(
```

Comment:
using msw and react query is all new to me, but I'm wondering if it might be better to set up the server somehwere more global so that it is accesible for a Storybook story or other use cases. Regardless, awesome job using all these technologies, I'm impressed!